.gitignore

```diff
@@ -76,7 +76,7 @@ packages/sdk/test-utils
 
 # Editor/IDE
 .idea
-.vscode
+.vscode/*
```

Comment:
This was actually "wrong", the `settings.json` was still ignored even though it's marked as not ignored.